odoo/tests/common.py

```diff
@@ -758,6 +755,19 @@ def _spawn_chrome(self, cmd):
             return
         pid = os.fork()
         if pid != 0:
+            for _ in range(25):
```

Comment:
Let's hope that 2.5 sec is enough.
Under some heavy load, it may takes more time.
the `_json_command` methods uses a timeout of 3s.